perf/istio-install/setup_istio.sh

```diff
@@ -108,7 +108,7 @@ function install_extras() {
   # Redeploy, this time with the Prometheus resource created
   helm template --set domain="${domain}" "${WD}/base" | kubectl apply -f -
   # Also deploy relevant ServiceMonitors
-  "${release}/bin/istioctl" manifest generate --set profile=empty --set addonComponents.prometheusOperator.enabled=true -d "${release}/manifests" | kubectl apply -f -
+  "${release}/bin/istioctl" manifest generate --set profile=empty -d "${release}/manifests" | kubectl apply -f -
```

Comment:
This does nothing here now, you can just remove it

Reply:
btw since the prometheusOperator charts are removed in the mixer removal plan, are we going to add that to samples, I think some of the servicemonitors are still valid, @douglas-reid ?

Reply:
i believe that @howardjohn has a PR that is doing precisely that (moving the service monitors to `samples/`).

Reply:
Yep. Fair credit to the comments - they were before my PR :slightly_smiling_face: